presto-docs/src/main/sphinx/release/release-348.md

```diff
@@ -0,0 +1,81 @@
+# Release 348 (XX Dec 2020)
+
+## General Changes
+
+* Add support for `DISTINCT` clause in aggregations within correlated subqueries. ({issue}`5904`)
+* Improve query performance by reducing worker to worker communication overhead. ({issue}`6126`)
+* Reduce memory pressure and improve performance of queries that contain join. ({issue}`6176`)
+* Fix ``EXPLAIN ANALYZE`` for certain queries that contain broadcast join. ({issue}`6115`)
+* Fix planning failures for queries that contain outer joins and aggregations using `FILTER (WHERE <condition>)` syntax. ({issue}`6141`)
+* Fix incorrect results when correlated subquery in join contains aggregation functions such as `array_agg` or `checksum`. ({issue}`6145`)
+* Fix duplicate call to event listener on query completion, if query failed early during preparation. ({issue}`6103`)
+
+## Web UI
+
+* Add support for OAuth2 authorization in Web UI. ({issue}`5355`)
+* Fix invalid operator stats being reported in stage performance view. ({issue}`6114`)
+
+## JDBC Driver Changes
+
+* Allow reading `timestamp with time zone` value as `ZonedDateTime` using `ResultSet.getObject(int column, Class<?> type)` method. ({issue}`307`)
+* Change representation of a `row` value. `ResultSet.getObject` now returns an instance of `io.prestosql.jdbc.Row` class, which better represents
```

Comment:
Is this a breaking change for users of the JDBC driver?

Reply:
I think it is reasonable to mark the bullet with something like `Warning: incompatible change` or `Warning: behavior change`. Not sure what wording should be used here.

Reply:
For the future: maybe we should have some graphical representation to mark notes which introduce backward incompatible changes?